File: attendancemanagement.client/src/services/api/types/result/AttendanceRecordsHistoryResult.ts
/** 勤怠更新履歴結果パラメータ */
export interface AttendanceRecordsHistoryResult {

    /** 履歴時刻（UTC）のUNIX時間（秒） */
    generateDateUtcUnixTimeSec: string;

    /** 更新理由 */
    reason: string;
}

File: Database/Accessor/Data.Accessor/Result/AttendanceRecordsHistoryResult.cs
namespace Data.Accessor
{
    /// <summary>
    /// 勤怠履歴データ
    /// </summary>
    public class AttendanceRecordsHistoryResult
    {
        /// <summary>
        /// 履歴登録日時
        /// </summary>
        public DateTime GenerateDateTime { get; set; }

        /// <summary>
        /// 更新理由
        /// </summary>
        public string Reason { get; set; } = string.Empty;
    }
}
